db-scripts/src/main/resources/cgds.sql

```diff
@@ -128,6 +128,15 @@ CREATE TABLE `cancer_study` (
   FOREIGN KEY (`TYPE_OF_CANCER_ID`) REFERENCES `type_of_cancer` (`TYPE_OF_CANCER_ID`)
 );
 
+-- --------------------------------------------------------
+CREATE TABLE `cancer_study_metadata` (
+  `CANCER_STUDY_ID` int(11) NOT NULL,
+  `CANCER_STUDY_IDENTIFIER` varchar(255),
```

Comment:
why reproduce this `CANCER_STUDY_IDENTIFIER` field here? Please remove it.

Reply:
@pieterlukasse My patch had this because I wasn't sure which way you would prefer, but it seems like you prefer joining this table with `cancer_study` in the query. (Because we get metadata by `CANCER_STUDY_IDENTIFIER` in the API). Either way works for me.